### PR TITLE
thud-test blog

### LINKED ISSUE #1137: Blog post: The thud test: Measuring OpenSearch documentation content
**Describe the blog post your would like to write**
The point is to describe the progress the documentation team has made in creating content for OpenSearch. I want our community to know the different areas of documentation that we've touched. The concept is around an old joke about 'the bigger the thud, the more documentation you have."  

**What is the title of the blog post?**
The thud test: Measuring OpenSearch documentation content

**Who are the authors?**
Heather Halter

**What is the proposed posting date?**
November 18 or later

### PR DESCRIPTION
**Pull request by hdhalter**
Signed-off-by: Heather Halter <hdhalter@amazon.com>

### Description
New blog post
 
### Issues Resolved
#1137

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by alicejw-aws — APPROVED** [review]
Looks good!

File: _authors/hdhalter.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Heather Halter
+short_name: hdhalter
+photo: '/assets/media/authors/hdhalter.jpg'
+linkedin: https://www.linkedin.com/in/heather-halter/
```
Comment (alicejw-aws) [inline _authors/hdhalter.markdown:5]: you can also include your github: <username> to link it here too

**Review by ariamarble — APPROVED** [review]
LGTM :)

File: _posts/_2022-11-08-thudtest.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+authors: 
+  - hdhalter
+comments: true
```
Comment (krisfreedain) [inline _posts/_2022-11-08-thudtest.md:5]: remove this line please

File: _posts/_2022-11-08-thudtest.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+title: "The thud test: Measuring OpenSearch Documentation"
+category:
```
Comment (krisfreedain) [inline _posts/_2022-11-08-thudtest.md:7]: insert line above this with the date we expect to publish - example:

date: 2022-11-10 01:01:01 -0700


Also "category" should be changed to "categories"

**Review by krisfreedain — CHANGES_REQUESTED** [review]
left a couple comments for necessary changes (there's one from Alice as well)

Also, the blog filename cannot start with an underscore - change from "_2022-11-08-thudtest.md"  to  "2022-11-08-thudtest.md"

**Comment by alicejw-aws** [comment]
update: we could not gain access rights to make the updates, so we recreated with Kris' updates in this PR: https://github.com/opensearch-project/project-website/pull/1147
Thanks

**Comment by krisfreedain** [comment]
closing per last comment - taking this forward with PR: https://github.com/opensearch-project/project-website/pull/1147

**Pull request closed without merging**